toolchain/parser/testdata/generics/interface/fail_missing_open_curly.carbon

```diff
@@ -14,8 +14,8 @@
 // CHECK:STDOUT: {kind: 'FileEnd', text: ''},
 // CHECK:STDOUT: ]
 
-// CHECK:STDERR: {{.*}}/toolchain/parser/testdata/generics/interface/fail_missing_open_curly.carbon:[[@LINE+1]]:15: Expected `{` to start interface definition.
+// CHECK:STDERR: {{.*}}/toolchain/parser/testdata/generics/interface/fail_missing_open_curly.carbon:[[@LINE+1]]:15: `interface` should either end with a `;` for a declaration or have a `{ ... }}` block for a definition.
```

Comment:
The opening `{` renders correctly here, but the double closing `}` in the definition still is doubled here?

Reply:
Oops, fixing